scorm.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+# https://www.ispringsolutions.com/blog/how-to-convert-html-to-scorm#turninghtmltoscormthemanualway
+# https://myelearningworld.com/3-best-ways-to-create-a-scorm-content-package/
+
+# Arguments
+
+outputFile=$1;
+outputTitle=$2;
+indexFile=$3;
+resources=${@:4};
+
+
+# Config
+
+scormPackageRoot='./scorm.package.tmp';
+scormPackageResourcesDir='resources';
+scormPackageTemplate='./scorm.package.template.tmp.zip';
+useScormDotComPackageTemplate=false;
+useScormDotComManifestTemplate=true;
+if [ -e $scormPackageRoot ]; then
+  echo "Error: $scormPackageRoot already exists.";
+  exit 1;
+fi;
+mkdir -p $scormPackageRoot/$scormPackageResourcesDir;
+if [ -e $outputFile ]; then
+  echo 'Warning: remove previous output file to renew it:';
+  rm -v $outputFile;
+fi;
+
+
+# Resources
+
+cp $indexFile $scormPackageRoot/$scormPackageResourcesDir/;
+for path in $resources; do
+  cp -r $path $scormPackageRoot/$scormPackageResourcesDir/;
+done;
+
+
+# Manifest
+
+## Structure and XSD
```

Comment:
TODO: Move to last SCORM version, actually SCORM 2004 3rd Edition, or, better move to the last standard, actually cmi5 — https://scorm.com/scorm-explained/business-of-scorm/scorm-versions/